.github/workflows/integration.yml

```diff
@@ -38,7 +38,13 @@ jobs:
 
       - name: Bring the stack up
         working-directory: ./ops
-        run: docker-compose up -d
+        run: |
+          ./scripts/stats.sh &
+          docker-compose up -d
+
+      - name: Wait for the Sequencer node
```

Comment:
This is the step that helps avoid memory issues. The problem was that the integration tests would start before the docker setup was finished starting. This doesn't fix /all/ memory issues but definitely fixes the most common issue. I having the stats.sh script will help debug future issues.